TechTalk.SpecFlow/Assist/TEHelpers.cs

```diff
@@ -119,29 +119,29 @@ private static void VerifyAllColumn(Table table, InstanceCreationOptions creatio
             }
         }
 
-        internal static IEnumerable<MemberHandler> GetMembersThatNeedToBeSet(Table table, Type type)
+        internal static IList<MemberHandler> GetMembersThatNeedToBeSet(Table table, Type type)

```

Comment:
```suggestion
        internal static List<MemberHandler> GetMembersThatNeedToBeSet(Table table, Type type)

```
If it is internal, I'd prefer to return the actual type, as this will not need interface calls and thus be faster. (Plus less allocations if you enumerate it.)